cli/lsp/documents.rs

```diff
@@ -93,6 +93,31 @@ pub enum LanguageId {
 }
 
 impl LanguageId {
+  pub fn as_media_type(&self) -> MediaType {
+    match self {
+      LanguageId::JavaScript => MediaType::JavaScript,
+      LanguageId::Jsx => MediaType::Jsx,
+      LanguageId::TypeScript => MediaType::TypeScript,
+      LanguageId::Tsx => MediaType::Tsx,
+      LanguageId::Json => MediaType::Json,
+      LanguageId::JsonC => MediaType::Json,
+      LanguageId::Markdown | LanguageId::Unknown => MediaType::Unknown,
```

Comment:
Should `MediaType` be `MediaType::Markdown` for `LanguageId::Mardown`?

Reply:
`MediaType::Markdown` doesn't exist. I'm not sure it's worth it to add one (maybe once we get a bug for something).